examples/feature_selection/plot_select_from_model_diabetes.py

```diff
@@ -1,100 +1,127 @@
-importance = np.abs(clf.coef_)
-print(importance)
+lasso = LassoCV().fit(X, y)
+importance = np.abs(lasso.coef_)
+feature_names = np.array(diabetes.feature_names)
+plt.bar(height=importance, x=feature_names)
+plt.title("Feature importances via coefficients")
+plt.show()
 
 ##############################################################################
-# Select from the model features with the higest score
-# ---------------------------------------------------------
+# Selecting features based on importance
+# --------------------------------------
+#
+# Now we want to select the two features which are the most important according
+# to the coefficients. The :class:`~sklearn.feature_selection.SelectFromModel`
+# is meant just for that. :class:`~sklearn.feature_selection.SelectFromModel`
+# accepts a `threshold` parameter and will select the features whose importance
+# (defined by the coefficients) are above this threshold.
 #
-# Now we want to select the two features which are the most important.
-# SelectFromModel() allows for setting the threshold. Only the features with
-# the `coef_` higher than the threshold will remain. Here, we want to set the
-# threshold slightly above the third highest `coef_` calculated by LassoCV()
-# from our data.
+# Since we want to select only 2 features, we will set this threshold slightly
+# above the coefficient of third most important feature.
+from sklearn.feature_selection import SelectFromModel
 
-idx_third = importance.argsort()[-3]
-threshold = importance[idx_third] + 0.01
+threshold = np.sort(importance)[-3] + 0.01
 
-idx_features = (-importance).argsort()[:2]
-name_features = np.array(feature_names)[idx_features]
-print('Selected features: {}'.format(name_features))
+sfm = SelectFromModel(lasso, threshold=threshold).fit(X, y)
+print("Features selected by SelectFromModel: "
+      f"{feature_names[sfm.get_support()]}")
 
-sfm = SelectFromModel(clf, threshold=threshold)
-sfm.fit(X, y)
-X_transform = sfm.transform(X)
+##############################################################################
+# Selecting features with Sequential Feature Selection
+# ----------------------------------------------------
+#
+# Another way of selecting features is to use Sequential Feature Selection
+# (SFS). SFS is a greedy procedure where, at each iteration, we choose the best
+# new feature to add to our selected features based a cross-validation score.
+# That is, we start with 0 features and choose the best single feature with the
+# highest score. The procedure is repeated until we reach the desired number of
+# selected features.
+#
+# We can also go in the reverse direction (backward SFS), *i.e.* start with all
+# the features and greedily choose features to remove one by one. We illustrate
+# both approaches here.
+
+from sklearn.feature_selection import SequentialFeatureSelector
 
-n_features = sfm.transform(X).shape[1]
+sfs_forward = SequentialFeatureSelector(lasso, n_features_to_select=2,
```

Comment:
we don't support `%time` magic or similar, do we?

I'd rather not compute the time manually in the example because this adds noise and this isn't the point of the example

Reply:
I think we actually do that quite a bit in examples and it might be informative?
```
~/checkout/scikit-learn/examples (master)
$ git grep "time()" | wc -l
257
```
but either way is fine with me.

Reply:
OK, I added it